ecosystem/sep-0006.md

```diff
@@ -59,11 +59,19 @@ The JWT should be included in all requests as request header:
 Authorization: Bearer <JWT>
 ```
 
-It is highly recommended to require authentication for all endpoints other than `/info` (and optionally `/fee`).
+It is highly recommended to require authentication for all endpoints other than `/info` (and optionally `/fee`). Some functionality, including the [Shared Accounts](#shared-omnibus-or-pooled-accounts) functionality described in the section below, may only be possible through the use of SEP-10.
+
+### Shared, Omnibus, or Pooled Accounts
+
+Client applications can use a single Stellar account to hold multiple users' funds. To distinguish users that use the same Stellar account, the decoded SEP-10 JWT's `sub` value may contain a memo value after the Stellar account (`G...:2810101841641761712`) OR the `sub` value will be a [Muxed Account (`M...`)](../core/cap-0027.md). The anchor should use the `sub` attribute in their data model to identify unique users.
+
+This document will refer to these accounts as shared accounts. See the [SEP-10 Memos](sep-0010.md#memos) and [Muxed Accounts](sep-0010.md#muxed-accounts) sections for more information.
+
+Note that Stellar accounts are either shared or they are not. This means anchors should ensure that a Stellar account previously authenticated with a memo should not be authenticated later without a memo. Conversely, an account that was previously authenticated without a memo should not be later authenticated as a shared account.
```

Comment:
I think this can get a bit confusing. What does it mean if an account that was previously indicated as shared authenticates without a memo? I don't see a use case for allowing this case, even if we treat the account without a memo as a distinct identity.

If we do come across a use case, we can update the SEP to be more prescriptive, but I'd vote to leave it general for now.

Reply:
OK, sounds good to me.